app/cdap/components/RecipeList/ViewAllRecipes.tsx

```diff
@@ -14,24 +14,28 @@
  * the License.
  */
 
-import React from 'react';
+import React, { useState } from 'react';
 import T from 'i18n-react';
 import RecipeList from 'components/RecipeList';
-
 import Box from '@material-ui/core/Box';
 import { SortBy, SortOrder } from './types';
 import Breadcrumbs from '@material-ui/core/Breadcrumbs';
 import { Link } from 'react-router-dom';
 import Typography from '@material-ui/core/Typography';
 import { getCurrentNamespace } from 'services/NamespaceStore';
+import RecipeDetails from 'components/RecipeDetails';
+import { IRecipeItem } from 'components/RecipeDetails';
 
 const PREFIX = 'features.WranglerNewUI.Recipe';
 
 const redirectToObj = `/ns/${getCurrentNamespace()}/wrangle`;
 
 const ViewAllRecipies = () => {
+  const [showRecipeDetails, setShowRecipedetails] = useState(false);
+  const [recipeDetails, setRecipeDetails] = useState<IRecipeItem>();
```

Comment:
this should better be recipe and setRecipe as details are just part of a recipe which is an entity with full details

Reply:
Done